Habicus/Habicus-Core
# HABI-8 Adding Persistence Store for Dev Environment

![](https://camo.githubusercontent.com/cf373500dfe17773807191f7fb730ea7ea4b959d/68747470733a2f2f692e696d6775722e636f6d2f5878585a6b6d4f2e706e67)

## Review Process:

##### Core Reviewers:
@austinsorrells 
@mweser 

##### Merge Owner:
@mweser
@Habicus/core-team
@austinsorrells
@schachte

## Status
**Ready**

## Description
This loads user test data from static input file and marshals it into the embedded persistence store. 

## Completed The Following (If Relevant)

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. Start the server
2. Login to the database at `localhost:7777/h2`
3. Login with admin/admin
4. Run a db query on * for the User table to verify results match the input file
